### DISCUSSION TITLE
Lock Management is not implemented yet?

### DISCUSSION THREAD

**Discussion by danilkorotkov** [HomeSpan/HomeSpan#39] [Q&A] [answered]
I didnt find it in Services.h
![Снимок экрана 2021-02-15 в 12 52 48](https://user-images.githubusercontent.com/25537053/107931047-d84d9600-6f8c-11eb-8996-3c5a2f8b3409.png)

**Comment by HomeSpan** [comment, marked as answer]
Lock Management uses a special type of data that is not implemented in HomeSpan.  See https://github.com/HomeSpan/HomeSpan/blob/master/docs/ServiceList.md.
